ide/coqide/coqide.ml

```diff
@@ -1103,6 +1118,8 @@ let build_ui () =
         radio "Set diff" 1 ~label:"Show diffs: only _added";
         radio "Set removed diff" 2 ~label:"Show diffs: added and _removed";
       ];
+    item "Show Proof Diffs" ~label:"_Show Proof with Diffs" ~accel:"<Shift><Alt>S"
```

Comment:
To have the same modifier as for the other display item, you should use `modifier_for_display#get` to get the modifier for the `accel` argument.

Reply:
The accelerators are a mess.  If you search for `~accel` in this file, you'll see that more than a few have explict tags like `<Alt>`.  There's some code that sets the tags for other items in the View menu, but it's not general, it doesn't seem to handle `Show Proof Diffs` (or the radio buttons just above).  However, changing the default in the Preferences menu does change the tags for `Show Proof Diffs`.

I'll make this
```    item "Show Proof Diffs" ~label:"_Show Proof with Diffs" ~accel:(modifier_for_display#get) ^ "S"```, but it's the only accelerator that explicitly has "modifier_for_display".

`Zoom out` is mapped to `Shift+Alt+-`, which can't be entered at least on my keyboard because it's not a shifted character (+ is).  FWIW, `modifier_for_display` should be called `modifier_for_view` to be consistent with similar routines named for the various menus.

Are you on a Mac?  Does it already give you a sensible sequence before this change?